package.json

```diff
@@ -92,5 +92,6 @@
     "debug": "node -r ts-node/register --inspect-brk ts-json-schema-generator.ts",
     "run": "ts-node-transpile-only ts-json-schema-generator.ts",
     "release": "yarn build && auto shipit"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
I added this because my default package manager is yarn 3 but I use node corepack. Without this yarn tried to migrate the repo to yarn 3.

I left it in because their may be others in my situation.

Reply:
Looks good.